miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/ProjectContactDto.java

```diff
@@ -0,0 +1,60 @@
+package uk.ac.bbsrc.tgac.miso.dto;
+
+public class ProjectContactDto {
+  private long projectId;
+  private long contactId;
+  private long contactRoleId;
+  // fields required for the case where a new contact is made
```

Comment:
When a new contact is made, these fields are needed to be stored in order to make the new contact through `saveNewContact` function in `DefaultProjectService`. Before, when using a contactDto, we already had the fields, so this is just to mimic that.